ui/admin/app/routes/scopes/scope/workers.js

```diff
@@ -1,10 +1,17 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
+import { loading } from 'ember-loading';
+import { confirm } from 'core/decorators/confirm';
+import { notifySuccess, notifyError } from 'core/decorators/notify';
+
 
 export default class ScopesScopeWorkersRoute extends Route {
   // =services
 
+  @service intl;
+  @service can;
+  @service notify;
```

Comment:
I don't think these are needed anymore after the merge.

Does the `notify` service needed to be injected if we're importing the `notifySuccess` and `notifyError` decorators?